pkgs/development/libraries/libutp/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake}:
+
+stdenv.mkDerivation rec {
+  pname = "libutp";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "transmission";
```

```suggestion
    owner = "bittorrent";
```

There are no changes in transmission so far.